solcjs

```diff
@@ -15,17 +15,39 @@ process.removeAllListeners('uncaughtException');
 var commander = require('commander');
 
 const program = new commander.Command();
+const parseOptimizeRuns = function(value) {
+  const parsedValue = parseInt(value, 10);
+  if (isNaN(parsedValue)) {
+    throw new commander.InvalidArgumentError('The "runs" setting must be an unsigned number');
+  }
+  return parsedValue;
+};
+
 program.name('solcjs');
 program.version(solc.version());
 program
   .option('--version', 'Show version and exit.')
   .option('--optimize', 'Enable bytecode optimizer.', false)
+  .option(
+    '--optimize-runs <optimize-runs>',
+    'The number of runs specifies roughly how often each opcode of the deployed code will be executed across the lifetime of the contract. ' +
+    'Lower values will optimize more for initial deployment cost, higher values will optimize more for high-frequency usage.',
+    parseOptimizeRuns
+  )
   .option('--bin', 'Binary of the contracts in hex.')
   .option('--abi', 'ABI of the contracts.')
   .option('--standard-json', 'Turn on Standard JSON Input / Output mode.')
   .option('--base-path <path>', 'Automatically resolve all imports inside the given path.')
   .option('-o, --output-dir <output-directory>', 'Output directory for the contracts.');
-program.parse(process.argv);
+try {
+  program.parse(process.argv);
+} catch(e) {
+  if(e instanceof commander.InvalidArgumentError) {
```

Comment:
Is all this really needed? According to the [documentation](https://www.npmjs.com/package/commander#override-exit-and-output-handling) it should just abort itself.

Reply:
Updated